packages/web-components/fast-components/src/menu-item/menu-item.styles.ts

```diff
@@ -260,7 +261,7 @@ export const menuItemStyles: (
         width: 100%;
         height: 100%;
         display: block;
-        fill: ${neutralForegroundRest};
+        fill: ${foregroundOnAccentRest};
```

Comment:
The fix I made in the Fluent update, and the preferred model we have, is to set `fill` to `currentcolor` when it's intended to match text. Ideally we set that as high as possible and avoid a bunch of `fill` attributes that are no longer necessary.